### ISSUE TITLE
MP JWT 1.2: Add TCK tests

### ISSUE THREAD

**Issue by teddyjtorres** [OpenLiberty/open-liberty#13580]
Create new project for the MP JWT 1.2 TCK tests.

**Comment by ayoho** [comment]
Most of the TCK tests are delivered and up and running. There are only 6 or so test classes left that are currently disabled because we haven't finished implementing the spec yet.

**Comment by clarkek123** [comment]
The MP JWT 1.2 TCK tests have been included in the new FAT - io.openliberty.microprofile.jwt.1.2.internal_fat_tc

The tests consist of the following and are running the latest level available which is MP JWT TCK 1.2-RC2 from Microprofile Eclipse.
```
ECPublicKeyAsJWKLocationTest 
ECPublicKeyAsPEMLocationTest 
ECPublicKeyAsPEMTest 
IssValidationFailTest 
IssValidationTest 
PublicKeyAsBase64JWKTest 
PublicKeyAsFileLocationURLTest 
PublicKeyAsJWKLocationTest 
PublicKeyAsJWKLocationURLTest 
PublicKeyAsJWKSLocationTest 
PublicKeyAsJWKSTest 
PublicKeyAsJWKTest 
PublicKeyAsPEMLocationTest
PublicKeyAsPEMLocationURLTest 
PublicKeyAsPEMTest 
TokenAsCookieIgnoredTest 
TokenAsCookieTest 
PrivateKeyAsJWKClasspathTest 
PrivateKeyAsPEMClasspathTest 
PrivateKeyAsJWKSClasspathTest 
ApplicationScopedInjectionTest 
AudArrayValidationTest 
AudValidationBadAudTest 
AudValidationMissingAudTest 
AudValidationTest 
ClaimValueInjectionTest 
CookieTokenTest
EmptyTokenTest 
InvalidTokenTest 
JsonValueInjectionTest 
PrimitiveInjectionTest 
PrincipalInjectionTest
ProviderInjectionTest 
RequiredClaimsTest 
RolesAllowedTest
RsaKeySignatureTest
UnsecuredPingTest 
PrivateKeyEndpoint 
RolesAllowedSignEncryptTest 
TokenUtilsEncryptTest 
TokenUtilsSignEncryptTest 
TokenUtilsExtraTest 
TokenUtilsTest 
```

**Comment by ayoho** [comment]
All TCK tests are delivered and using the final version of the API, so I'm going to close this issue. The only remaining issue is https://github.com/OpenLiberty/open-liberty/issues/14732, which we're tracking directly underneath the overall epic (#11022).
